### PR TITLE
Test content

### PR DESCRIPTION
**Pull request by davidmohringmrf**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,10 +11,9 @@
     "url": "git+https://github.com/Marfeel/flowcards-playground.git"
   },
   "scripts": {
-    "test:local": "npm run appium:tab && END_TO_END_MODE=local npm t",
-    "test:remote-local": "END_TO_END_MODE='browserstack-local' npm t",
-    "test:remote": "END_TO_END_MODE='browserstack-remote' npm t",
-    "test": "wdio conf/conf.js",
+    "test:local": "END_TO_END_MODE=local wdio conf/conf.js",
+    "test:remote-local": "END_TO_END_MODE=browserstack-local wdio conf/conf.js",
+    "test:remote": "END_TO_END_MODE=browserstack-remote wdio conf/local.conf.js",
```
Comment (oscard0m) [inline package.json]: 💅🏽 I created command `test` to """centralize""" the command in one place so we can apply flags to each of them but if you prefer it like this is fine to me.

File: package.json [hunk 2]
```diff
@@ -11,10 +11,9 @@
     "url": "git+https://github.com/Marfeel/flowcards-playground.git"
   },
   "scripts": {
-    "test:local": "npm run appium:tab && END_TO_END_MODE=local npm t",
-    "test:remote-local": "END_TO_END_MODE='browserstack-local' npm t",
-    "test:remote": "END_TO_END_MODE='browserstack-remote' npm t",
-    "test": "wdio conf/conf.js",
+    "test:local": "END_TO_END_MODE=local wdio conf/conf.js",
+    "test:remote-local": "END_TO_END_MODE=browserstack-local wdio conf/conf.js",
```
Comment (oscard0m) [inline package.json]:
```suggestion
```

File: package.json (hunk 1 above)
Comment (oscard0m) [inline package.json]:
```suggestion
    "test:browserstack": "END_TO_END_MODE=browserstack wdio conf/local.conf.js",
```

File: tests/utils/cardContent.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    const contentInfo = await browser.executeAsync(async (cardSelector, done) => {
+        let amp = document.querySelector(`${cardSelector} article`).firstElementChild.shadowRoot.AMP
+        let info = {url: amp.url, title: amp.title};
+        done(info)
+    }, cardSelector); 
+
+    return contentInfo
+}
+
+const hasRightContentLoaded = async (browser, cardSelector, contentConfig) => {
```
Comment (oscard0m) [inline tests/utils/cardContent.js]: 💅🏽  To not confuse with left-right, what do you think about this name?

```suggestion
const isPageContentLoaded = async (browser, cardSelector, contentConfig) => {
```

File: tests/utils/cardContent.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    const expectedUrl = contentConfig.url.includes(playgroundUrlPlaceholders) ? 
+    contentConfig.url.replace(playgroundUrlPlaceholders, '') :
+    contentConfig.url;
+    
+    const contentInfo = await getCardAMPContent(browser, cardSelector);
```
Comment (oscard0m) [inline tests/utils/cardContent.js:19]: why is contentInfo coupled to AMP? Should this method work for web and AMP?

File: tests/utils/snapPoints.js [hunk 5]
```diff
@@ -39,7 +39,32 @@ const isAtPercentageSnapPoint = async (browser, cardSelector, expectedSnapPointP
     return true
 }
 
+const isAtSnapPoint = async (browser, cardSelector, snapPointValue)=>{
```
Comment (oscard0m) [inline tests/utils/snapPoints.js:42]: 💅🏽  What do you think about this naming?
* `verifyCardPosition` 
* `verifyCardRelativePosition` (percentage)
* `verifyCardAbsolutePosition` (number)

File: tests/utils/snapPoints.js [hunk 6]
```diff
@@ -39,7 +39,32 @@ const isAtPercentageSnapPoint = async (browser, cardSelector, expectedSnapPointP
...
+
+    return isAtSnapPoint;
+}
+
 module.exports = {
-    isAtPercentageSnapPoint,
-    isAtAbsoluteSnapPoint
+    isAtSnapPoint,
+    checkAbsoluteSnapPoint,
+    checkPercentageSnapPoint
```
Comment (oscard0m) [inline tests/utils/snapPoints.js]: Is there a reason to expose this methods? Should we only expose the one choosing the strategy instead?

File: tests/utils/cardContent.js (hunk 4 above)
Comment (davidmohringmrf) [reply tests/utils/cardContent.js:19]: the card content is always an AMPDocument

**Pull request merged**